programs/stake/src/stake_state.rs

```diff
@@ -384,21 +398,28 @@ impl Stake {
         point_value: &PointValue,
         vote_state: &VoteState,
         stake_history: Option<&StakeHistory>,
+        inflation_point_calc_tracer: &mut Option<impl FnMut(&InflationPointCalculationEvent)>,
```

Comment:
As a decent usecase of https://github.com/rust-lang/rust/issues/63063, I want to define a type alias for this....